superset-frontend/src/components/Button/index.tsx

```diff
@@ -233,22 +233,23 @@ const SupersetButton = styled(BootstrapButton)`
   }
 `;
 
-export default function Button(props: ButtonProps) {
-  const buttonProps = {
-    ...props,
-    bsSize: props.buttonSize,
-    disabled: props.disabled || false,
-    placement: props.placement || 'top',
-  };
-  const { tooltip, placement, dropdownItems } = props;
-  delete buttonProps.tooltip;
-  delete buttonProps.placement;
```

Comment:
TypeScript will now complaint https://devblogs.microsoft.com/typescript/announcing-typescript-4-0-beta/#operands-for-delete-must-be-optional

I refactored the whole button component with prop deconstruction to eliminate the need to use `delete` prop.